frb_codegen/src/config.rs

```diff
@@ -79,7 +80,7 @@ pub struct Opts {
     pub manifest_path: String,
     pub dart_root: Option<String>,
     pub build_runner: bool,
-    pub block_index: usize, //NOTE: the index is counted from 1, not 0
+    pub block_index: BlockIndex, //NOTE: the index is counted from 1, not 0
```

Comment:
remove the outdated comments: "//NOTE: the index is counted from 1, not 0"

Reply:
> remove the outdated comments: "//NOTE: the index is counted from 1, not 0"

Fixed in https://github.com/fzyzcjy/flutter_rust_bridge/pull/527/commits/cc7cfbc4f1a0b944bff5d25383e8f87edc1a914f.